test/sharness/Makefile

```diff
@@ -16,6 +16,7 @@ all: clean deps $(T) aggregate
 clean:
 	@echo "*** $@ ***"
 	-rm -rf test-results
+	-rm -rf bin/ipfs
```

Comment:
So this: https://github.com/jbenet/go-ipfs/blob/feat/repo-fsrepo/test/sharness/Makefile#L35-L37  isn't working? can you repro easily?